roles/php/tasks/main.yml

```diff
@@ -34,30 +34,30 @@
       - composer
     state: present
 
-- name: check for presence of the remi 73 repository file
+- name: check for presence of the remi 72 repository file
   stat:
-    path: "/etc/yum.repos.d/remi-php73.repo"
-  register: remi_php73_config
+    path: "/etc/yum.repos.d/remi-php72.repo"
+  register: remi_php72_config
 
```

Comment:
I don't think we need to register the presence of this file. The next copy task will take care of that automatically